scripts/dev.sh

```
@@ -110,7 +116,7 @@ find_server_processes() {
         if [[ -n "$line" ]]; then
             local pid=$(echo "$line" | awk '{print $2}')
             # Add to pids if not already present
-            if [[ ! " ${pids[@]} " =~ " $pid " ]]; then
+            if [[ ! " ${pids[@]:-} " =~ " $pid " ]]; then
```

Shellcheck (0.11.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)